src/main.rs

```diff
@@ -135,7 +135,11 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     let shard = SnapchainShard::new(0); // Single shard for now
     let validator_address = Address(keypair.public().to_bytes());
-    let validator = SnapchainValidator::new(shard.clone(), keypair.public().clone());
+    let validator = SnapchainValidator::new(
+        shard.clone(),
+        keypair.public().clone(),
+        Some(app_config.rpc_address.clone()),
```

Comment:
Should this be an option?

Reply:
Made it an option because I didn't want to store an rpc address on the validator running inside this node.